packages/dd-trace/test/appsec/blocking.spec.js

```diff
@@ -277,3 +248,41 @@ describe('blocking', () => {
     })
   })
 })
+
+describe('waf actions', () => {
+  const blocking = require('../../src/appsec/blocking')
+
+  it('get block_request as blocking action', () => {
+    const blockRequestActionParameters = {
+      status_code: 401,
+      type: 'html'
+    }
+    const actions = {
+      block_request: blockRequestActionParameters
+    }
+    expect(blocking.getBlockingAction(actions), blockRequestActionParameters)
```

Comment:
uuuh are you sure that's the correct syntax for expect ? does that second argument do anything ?

Reply:
Changed to correct expect syntax